intel_extension_for_transformers/neural_chat/examples/deployment/rag/askdoc.yaml

```diff
@@ -30,9 +30,12 @@ retrieval:
     enable: true
     args:
         input_path: "./askdoc_docs"
-        persist_directory: "./askdoc_persist"
```

Comment:
must be persist_directory since RAG requires

Reply:
This has been tested on SPR server. 'persist_directory' is not currently compatible with RAG. 
The RAG service could only be started correctly using 'persist_dir', otherwise error ocurred.

Reply:
Changed back persist_directory.

Reply:
Verified on trial demo. Details needs to check with xuhui.